content/doc/book/blueocean/index.adoc

```diff
@@ -13,33 +13,56 @@ ifdef::backend-html5[]
 endif::[]
 
 
+
 [blue-ocean]
 = Blue Ocean
 
-This chapter will cover all aspects of Blue Ocean, from the Dashboard, to
-viewing branches and results from individual Pipeline runs, to using the Visual
-Editor to modify Pipelines as code.
+This chapter covers all aspects of Blue Ocean's functionality, including how to:
+
+* <<blueocean/getting-started#,get started with Blue Ocean>> - covers how to set
+  up Blue Ocean in Jenkins and access the Blue Ocean interface,
+* <<blueocean/creating-pipelines#,create a new Pipeline>> project,
+* use the <<blueocean/dashboard#,Dashboard>>,
+* use the <<blueocean/activity#,Activity view>> - where you can access
+  <<blueocean/activity#activity,lists of your current and previously completed
+  Pipeline/item runs>>, as well as your Pipeline project's
+  <<blueocean/activity#branches,branches>> and any opened
+  <<blueocean/activity#pull-requests,Pull Requests>>,
+* use the <<blueocean/pipeline-run-details#,Pipeline run details view>> - where
+  you can access details (i.e. the console output) for a particular
+  Pipeline/item run, and
+* use the <<blueocean/pipeline-editor#,Pipeline Editor>> to modify Pipelines as
+  code, which are committed to source control.
+
+This chapter is intended for Jenkins users of all skill levels, but beginners
+may need to refer to some sections of the <<pipeline#,Pipeline>> chapter to
+understand some topics covered in this Blue Ocean chapter.
 
 For an overview of content in the Jenkins User Handbook, see
 <<getting-started#,User Handbook Overview>>.
 
+
 [[overview]]
 == What is Blue Ocean?
 
-Blue Ocean rethinks the user experience of Jenkins.  Designed from the ground
-up for <<pipeline#, Jenkins Pipeline>>, but still compatible with Freestyle
-jobs, Blue Ocean reduces clutter and increases clarity for every member of the
-team.
+Blue Ocean rethinks the user experience of Jenkins. Designed from the ground up
+for link:../pipeline/[Jenkins Pipeline], but still compatible with freestyle jobs,
```

Comment:
Isn't "Freestyle" like "Pipeline" here?

Reply:
Did you mean in terms of capitalization?

Reply:
Yes.